src/GeneralRules/ReExcessiveArgumentsRule.class.st

```diff
@@ -30,6 +30,12 @@ ReExcessiveArgumentsRule >> argumentsCount [
 
 { #category : #running }
 ReExcessiveArgumentsRule >> basicCheck: aMethod [
+
+	"If method includes UFFI calls then it is OK to have high number of arguments"
+	(aMethod ast allChildren anySatisfy: [:node | 
```

Comment:
There is #isFFIMethod

Reply:
@dionisiydk Oh fine - lets handle this in https://github.com/pharo-project/pharo/pull/7789